packages/twenty-front/src/modules/page-layout/widgets/widget-card/components/WidgetCardContent.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled';
+import { type ReactNode } from 'react';
+
+import { type WidgetCardContext } from '../types/WidgetCardContext';
+
+export type WidgetCardContentProps = {
+  children?: ReactNode;
+  widgetCardContext?: WidgetCardContext;
+  className?: string;
+};
+
+const StyledWidgetCardContent = styled.div<{
+  widgetCardContext?: WidgetCardContext;
+}>`
+  align-items: center;
+  display: flex;
+  height: 100%;
+  width: 100%;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: ${({ theme }) => theme.spacing(2)};
+
+  ${({ theme, widgetCardContext }) => {
+    switch (widgetCardContext) {
+      case 'recordPage':
+        return `
+          border: 1px solid ${theme.border.color.medium};
+          border-radius: ${theme.border.radius.md};
+        `;
+
+      default:
+        return '';
+    }
+  }}
+`;
+
+export const WidgetCardContent = ({
```

Comment:
If the `WidgetCardContent` only re-injects props to the `StyledWidgetCardContent` component, we could do:

```ts
export { StyledWidgetCardContent as WidgetCardContent }
```